packages/cli/src/ceramic-cli-utils.ts

```diff
@@ -210,8 +213,8 @@ export class CeramicCliUtils {
       if (pubsubTopic) {
         config.network.pubsubTopic = pubsubTopic
       }
-      if (gateway) {
-        config.node.gateway = gateway
+      if (gateway || readOnly) {
+        config.node.gateway = gateway || readOnly
```

Comment:
can we rename the value within ceramic core too while we're at it?

Reply:
gotcha